spark-screenshot-testing/src/test/kotlin/com/adevinta/spark/icons/AnimatedIconsScreenshot.kt

```diff
@@ -40,44 +40,144 @@ internal class AnimatedIconsScreenshot {
     @get:Rule
     val paparazzi = paparazziRule()
 
+//    Blocked until https://github.com/cashapp/paparazzi/pull/1645 is merged and available
+//    @Test
+//    fun bellShake() {
+//        val view = paparazzi.gifView {
+//            Icon(
+//                sparkIcon = SparkAnimatedIcons.bellShake(),
+//                contentDescription = "Bell Shake Animation",
+//                size = IconSize.ExtraLarge,
+//                tint = Color.Black,
+//                atEnd = true,
+//            )
+//        }
+//
+//        paparazzi.gif(view, start = 0, end = 800, fps = 60)
+//    }
+
     @Test
-    fun bellShake() {
+    fun collapseExpand() {
```

Comment:
I feel like this entire test class could be simplified with a parameterized test, with icon/description/size/start/end as inputs.

Reply:
I created it as a good first issue https://github.com/leboncoin/spark-android/issues/1721 if any contributors want to do it until i get more free time